woocommerce/woocommerce-android
# [Dynamic Dashboard] Align top performers and stats cards

### Description
This PR adds two changes:
1. It introduces a common component to be used as the header of the stats and top performers cards, to make sure they stay consistent.
2. It updates the date formatting in the top performer's card to align with the stats and what iOS uses.

### Testing instructions
1. Open the dashboard.
2. Make sure the Performance and Top Performers cards are enabled.
3. Confirm their header look and behave as expected.

### Images/gif
